src/lm.jl

```diff
@@ -243,3 +243,27 @@ function confint(obj::LinearModel; level::Real=0.95)
     hcat(coef(obj),coef(obj)) + stderror(obj) *
     quantile(TDist(dof_residual(obj)), (1. - level)/2.) * [1. -1.]
 end
+
+"""
+    cooksdistance(obj::LinearModel)
+
+Compute [Cook's distance](https://en.wikipedia.org/wiki/Cook%27s_distance)
+for each observation in linear model `obj`, giving an estimate of the influence
+of each data point.
+Currently only implemented for LinearModel models without weights.
+"""
+function StatsBase.cooksdistance(obj::LinearModel)
+    u = residuals(obj)
+    mse = dispersion(obj,true)
+    k = dof(obj)-1
+    d_res = dof_residual(obj)
+    X = modelmatrix(obj)
```

Comment:
Use `crossmodelmatrix` here, then you get `X` and `X'X` back, but `X'X` has been wrapped in `Symmetric`, which allows the linear algebra methods below to be more specialized.

Reply:
done